libraries/GCS_MAVLink/GCS.h

```diff
@@ -595,8 +600,14 @@ class GCS_MAVLINK
     virtual uint8_t high_latency_wind_speed() const { return 0; }
     virtual uint8_t high_latency_wind_direction() const { return 0; }
     int8_t high_latency_air_temperature() const;
-#endif // HAL_HIGH_LATENCY2_ENABLED
 
+    MAV_RESULT handle_control_high_latency(const mavlink_command_long_t &packet);
+
+    // true if this is a high latency link
+    bool is_high_latency_link = false;
```

Comment:
don't init class vars to false, wastes flash

Reply:
Have now removed var inits